client/components/external-link/README.md

```diff
@@ -12,7 +12,15 @@ import ExternalLink from 'components/external-link';
 
 class MyComponent extends React.Component {
 	render() {
-		return <ExternalLink icon={ true } href="https://wordpress.org" onClick="somefunction()">WordPress.org</ExternalLink>;
+		return (
+			<ExternalLink
+				icon={ true }
```

Comment:
Optional: I know this was here before, but since we're updating it, this is a great occasion to improve it:
```suggestion
				icon
```